src/lib/i18n.json

```diff
@@ -316,6 +317,13 @@
     "azure-client-get-build-client-err": "Could not get build client. Check the Azure DevOps credential",
     "azure-client-get-task-agent-client-err": "Could not get task agent client. Check the Azure DevOps credential.",
 
+    "azure-key-vault-missing-name": "Missing key vault name. Provide it",
+    "azure-key-vault-missing-secret-name": "Missing key vault secret name. Provide it",
+    "azure-key-vault-missing-secret-value": "Missing key vault secret value. Provide it",
+    "azure-key-vault-client-err": "Could not get Azure key vault client. Check your DevOps credentials.",
```

Comment:
Minor rewording:

`"azure-key-vault-client-err": "Could not get Azure key vault client. Check your Azure credentials."`

In this case the credentials are for Azure and not for Azure DevOps.

Reply:
accept